packages/admin-next/dashboard/src/v2-routes/campaigns/common/components/create-campaign-form-fields/create-campaign-form-fields.tsx

```diff
@@ -98,33 +125,7 @@ export const CreateCampaignFormFields = ({ form, fieldScope = "" }) => {
           }}
         />
 
-        <Form.Field
-          control={form.control}
-          name={`${fieldScope}currency`}
-          render={({ field: { onChange, ref, ...field } }) => {
-            return (
-              <Form.Item>
-                <Form.Label>{t("fields.currency")}</Form.Label>
-                <Form.Control>
-                  <Select {...field} onValueChange={onChange}>
-                    <Select.Trigger ref={ref}>
-                      <Select.Value />
-                    </Select.Trigger>
-
-                    <Select.Content>
-                      {Object.values(currencies).map((currency) => (
-                        <Select.Item value={currency.code} key={currency.code}>
-                          {currency.name}
-                        </Select.Item>
-                      ))}
-                    </Select.Content>
-                  </Select>
-                </Form.Control>
-                <Form.ErrorMessage />
-              </Form.Item>
-            )
-          }}
-        />
+        <div></div>
```

Comment:
rm?
```suggestion
```

Reply:
This is actually needed to keep the grid spot the same size :D

Reply:
Ahh I see :smile